src/_component/experience/experience.pug

```diff
@@ -0,0 +1,78 @@
+            description: "I was spending fun time on creating amazing interfaces with React and GraphQL. I also mentored their junior and mid level staff into web development."
+        },
+        {
+            position: "Rect Developer",
+            logo: "asset/company-logo/big-telematics.png",
+            company: "Telematics",
+            description: "I was spending fun time on creating amazing interfaces with React and GraphQL. I also mentored their junior and mid level staff into web development."
+        },
+    ]
+
+
+.red-box
+    .section-title Experience
+.experience
+    .space(height="30")
+    .button-wrapper
+        button.youtube-button
+            a.youtube-link(href="https://www.youtube.com/watch?v=7lgg_MSblRE")      
+                span.youtube-icon(class=`icon-youtube`)
+    .space(height="30")
+
+    input.experience-input(type="radio" name="grouped-inputs" checked id="experience-default")
+    label.experience-description__close(for="experience-default")
+    ul.header-experience__list
+        each experience, key in experienceList
+            li.header-experience__item
+                label.header-experience-label(for=`experience-id-${key}`)
+                    img.header-experience__logo(src="https://via.placeholder.com/150")
+                    .header-experience__wrapper
+                        .header-experience__position=experience.position
+                        .header-experience__company=experience.company
+                    span.icon-arrow-down
+                input.experience-input(type="radio" name="grouped-inputs" id=`experience-id-${key}`)
+                .experience-description
+                    p I was spending fun time on creating amazing interfaces with React and GraphQL. I also mentored their junior and mid level staff into web development.
+
+
+
+
+
+    //- input.experience__input(type="radio", name="grouped-inputs", id="experience-default", checked)
+    //- lable.experience-close(for="experience-default")
+
+    //- ul.experience__content-wrapper
+    //-     each experience, key in experienceList
+    //-         li.experience__content-item
+    //-             input.experience__input(type="radio", name="grouped-inputs", id=`experience-id-${key}`)
+    //-             lable.experience-arrow(for=`experience-id-${key}`)
+    //-                 img.experience__logo(src="https://via.placeholder.com/150")
+    //-                 .experience-header__wrapper
+    //-                     .experience__position=experience.position
+    //-                     .experience__company=experience.company
+    //-                 span.icon-arrow-down
+    //-         .experience-description
+    //-             p.experience-text I was spending fun time on creating amazing interfaces with React and GraphQL. I also mentored their junior and mid level staff into web development.
+
+    .space(height="30")
+    
+    .arrow-down
+        span(class=`icon-arrow-down`)
```

Comment:
double quotes

Reply:
yeah, I forgot it...I encounter an error during the building the code and that's why it's
 not there :D